test/BuildingRegistry.Tests/BackOffice/Validators/PlanBuildingUnitRequestValidatorTests.cs

```diff
@@ -33,8 +34,8 @@ public void GivenInvalidBuildingId_ThenReturnsExpectedFailure(string buildingId)
             });
 
             result.ShouldHaveValidationErrorFor(nameof(PlanBuildingUnitRequest.GebouwId))
-                .WithErrorCode(ValidationErrorCodes.BuildingUnit.BuildingNotFound)
-                .WithErrorMessage(ValidationErrorMessages.BuildingUnit.BuildingInvalid(buildingId));
+                .WithErrorCode(ValidationErrors.Common.BuildingNotFound.Code)
```

Comment:
I don't like the tests using the codes instead of the actual error codes/messages. I actually see that this code was changed and if it was changed incorrectly, the test doesn't catch it.
![image](https://user-images.githubusercontent.com/18366653/216097010-8a917074-a807-4413-896a-17e768a8c0f0.png)